From 18338111f2f362cfc14b5a77ea44ac87e04ae4a9 Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <vital.had@gmail.com>
Date: Fri, 10 Oct 2025 13:54:40 +0800
Subject: [PATCH] udppktinfo.c: fix bug in a macro

Signature was changed in: https://github.com/krb5/krb5/commit/87b7d86676bff0b64a9eea39849abc97fdbe2972
But fallback was not updated to match it.
---
 src/lib/apputils/udppktinfo.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/apputils/udppktinfo.c b/src/lib/apputils/udppktinfo.c
index 8eed7a29d4c..0ecbccbe90c 100644
--- lib/apputils/udppktinfo.c
+++ lib/apputils/udppktinfo.c
@@ -363,7 +363,7 @@ set_msg_from_ip_sendsrcaddr(struct msghdr *msg, struct cmsghdr *cmsgptr,
 }
 
 #else /* HAVE_IP_PKTINFO || IP_SENDSRCADDR */
-#define set_msg_from_ipv4(m, c, f, l, a) EINVAL
+#define set_msg_from_ipv4(m, c, f, a) EINVAL
 #endif /* HAVE_IP_PKTINFO || IP_SENDSRCADDR */
 
 #ifdef HAVE_IPV6_PKTINFO
